glean-core/android/src/main/java/mozilla/telemetry/glean/private/HistogramBase.kt

```diff
@@ -22,3 +22,9 @@ interface HistogramBase {
      */
     fun accumulateSamples(samples: List<Long>)
 }
+
+// glean_parser template currently expects `HistogramMetricBase` as the name
+// and since this alias was defined in `service-glean` in android-components,
+// we need to keep the alias until the parser template is updated also.
+// TODO: See Bug ####### for more information.
```

Comment:
Fill in the bug number before landing, if you would

Reply:
Whoops, I forgot about this. Thanks for reminding me :)